shortcircuit-api/src/main/java/com/powsybl/shortcircuit/FaultResult.java

```diff
@@ -16,45 +18,87 @@
  * Results for one fault computation.
  *
  * @author Boubakeur Brahimi
+ * @author Coline Piloquet <coline.piloquet at rte-france.com>
  */
 public final class FaultResult extends AbstractExtendable<FaultResult> {
 
-    private final String id;
+    private final Fault fault;
 
-    private final double threePhaseFaultCurrent;
+    private final double threePhaseFaultActivePower;
```

Comment:
Naming:
maybe @JB-H has a better vision of what we want to put in here, but I think the right name would be "short circuit power". Actually it seems it has nothing to do with active power, it's computed by convention using nominal voltage.

From what I understand, it does not make sense either for non-symmetrical faults, so "three phase" is redundant.